axum-extra/src/extract/json_deserializer.rs

```diff
@@ -0,0 +1,446 @@
+use axum::async_trait;
+use axum::extract::{FromRequest, Request};
+use axum_core::__composite_rejection as composite_rejection;
+use axum_core::__define_rejection as define_rejection;
+use axum_core::extract::rejection::BytesRejection;
+use bytes::Bytes;
+use http::{header, HeaderMap};
+use serde::Deserialize;
+use std::marker::PhantomData;
+
+/// JSON Extractor for zero-copy deserialization.
+///
+/// Deserialize request bodies into some type that implements [`serde::Deserialize<'de>`].
```

Comment:
This link does not seem to work. See https://docs.rs/axum-extra/latest/axum_extra/extract/struct.JsonDeserializer.html

Reply:
I think we just forgot a reference for it like https://docs.rs/axum-extra/latest/src/axum_extra/extract/json_deserializer.rs.html#39. Do you wanna submit a PR perhaps?

Reply:
Yes I can do it.
I'm wondering why it is working for `serde::de::DeserializeOwned`. I'm not sure I understand how all of this work.

Reply:
Because that symbol has been imported.

Reply:
> Because that symbol has been imported.

I only see `use serde::Deserialize;`. I don't see that `serde::de::DeserializeOwned` is imported.

Reply:
Huh yeah thats true. No idea 😅

Reply:
I think it might be the `<'de>` that's confusing rustdoc. How about:

```suggestion
/// Deserialize request bodies into some type that implements [`serde::Deserialize<'de>`][serde::Deserialize].
```

Reply:
I confirm this is working.
https://github.com/tokio-rs/axum/pull/2498